client/src/components/PayBill.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useRef, useEffect, useState } from 'react';
+import $ from 'jquery';
+import ThankYou from './ThankYou.jsx';
+
+const payeeEmail = 'sb-dmus17722352@personal.example.com';
+const hostName = 'Milo';
+const billInfo = [
```

Comment:
This works great for this proof of concept, but would this data be coming from the App top-level state once we fully integrate this?

Reply:
Yes, I will link to APP level state to have them fully integrated